### PR TITLE
correction

### PR DESCRIPTION
**Pull request by schellhorn**

### CODE REVIEW

File: krr.bib [hunk 1]
```diff
@@ -6914,7 +6914,7 @@ @Article{elvhun93b
 }
 
 @Article{emdkow76a,
-  author =	 "M. H. van Emden and R. A. Kowalski",
+  author =	 "M. van Emden and R. Kowalski",
```
Comment (tortinator) [inline krr.bib]: composite names should better be in braces

File: krr.bib [hunk 2]
```diff
@@ -14909,6 +14911,7 @@ @Article{nilsson86
 
 @article{nilsson94a,
   author =	 "N. Nilsson",
+
```
Comment (tortinator) [inline krr.bib]: ???

File: krr.bib [hunk 3]
```diff
@@ -9330,6 +9330,7 @@ @Article{giomar94a
   pages =	 "161-179"
 }
 
+
```
Comment (tortinator) [inline krr.bib]: Why 2 newlines?

File: krr.bib [hunk 4]
```diff
@@ -12562,6 +12563,7 @@ @InProceedings{levesque84b
   publisher =	 "Morgan Kaufman"
 }
 
+
```
Comment (tortinator) [inline krr.bib]: ditto

File: krr.bib [hunk 5]
```diff
@@ -17217,6 +17219,7 @@ @InProceedings{schaub09a
   crossref =	 "lpnmr09"
 }
 
+
```
Comment (tortinator) [inline krr.bib]: ditto

File: krr.bib [hunk 6]
```diff
@@ -17797,7 +17800,6 @@ @Article{sebastiani07a
 @Article{seforewi02a,
   author =	 "A. Segre and S. Forman and G. Resta and A. Wildenberg",
   title =	 "Nagging: A scalable fault-tolerant paradigm for distributed search",
-
```
Comment (tortinator) [inline krr.bib]: Why a blank line?

File: krr.bib (hunk 1 above)
Comment (schellhorn) [reply krr.bib]: Ok, I will do so.

File: krr.bib (hunk 2 above)
Comment (schellhorn) [reply krr.bib]: I have no clue what's going on here... seems my editor adds randomly newlines -.-

**Comment by schellhorn** [comment]
Now it should be without newlines ... 
I did not correct composit names of other entries.

**Pull request merged**